modules/ROOT/pages/migrate-client-apps.adoc

```diff
@@ -0,0 +1,124 @@
+= Migrate Client Applications
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: migrate client applications, migrate applications, migrate 
+
+You might have application client credentials residing outside of Anypoint Platform if you were previously using a different API management solution. You must first migrate those client applications to Anypoint Platform, with their client ID and client secret credentials, before you can start using the credentials in Anypoint Platform. 
+
+Your client applications can consume APIs whose organization is associated to an Anypoint Platform client provider or an external client provider. The instructions for these two scenarios differ:
+
+* <<migrate_with_internal,Migrate client applications for APIs with Anypoint Platform client providers>>
+* <<migrate_with_external,Migrate client applications for APIs with external client providers>>
+
+== Prerequisites
+
+Before migrating the client application, ensure that you:
+
+* Obtain the required token and credentials:
+** Client ID and client secret for the client application to migrate
+** Anypoint Platform https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/access-management-api/minor/1.0/pages/Authentication/[token]
+* Review the <<considerations,scope and limitations>>
```

Comment:
```suggestion
* Review the <<considerations, scope, and limitations>>
```

Reply:
...but where are the considerations?

Reply:
Kim, this is the link. You can't see it in text.